build-system/tasks/presubmit-checks.js

```diff
@@ -59,6 +59,25 @@ var forbiddenTerms = {
       'test/functional/test-url-replacements.js'
     ],
   },
+  'installViewerService': {
+    message: privateServiceFactory,
+    whitelist: [
+      'src/amp-core-service.js',
+      'src/service/history-impl.js',
+      'src/service/viewer-impl.js',
+      'src/service/viewport-impl.js',
+      'test/functional/test-cid.js',
```

Comment:
/cc @erwinmombay is there a way for us to declare whitelists for the whole `test/` directory? I don't quite see a reason to be protective over this kind of uses in tests.